assets/css/structures/_definition-content.scss

```diff
@@ -57,12 +57,18 @@
     }
 
     & h2 ~ p,
-    & h2 ~ ul {
+    & h2 ~ ul,
+    & h2 ~ blockquote {
       font-size: inherit;
       // border-left: 0.1rem solid lightgrey;
       // padding-left: 1rem;
       grid-column: 2;
     }
+
+    & h2 ~ blockquote {
+      border-left: 0.1rem solid lightgrey;
+      padding-left: 1rem;
+    }
```

Comment:
I took the liberty of fixing the styling of blockquotes. They currently render in a very slim column (see the [optical allyship definition](https://www.selfdefined.app/definitions/optical-allyship/) for an example), so I styled them like this:

<img width="733" alt="Screen Shot 2020-11-17 at 11 18 05 AM" src="https://user-images.githubusercontent.com/3411183/99437120-e543e600-28c6-11eb-90dc-3ed2696c0d52.png">

Please let me know if you have any feedback or would rather fix this in a different PR.

Reply:
Oh yikes, thanks for catching and fixing this. This is perfect! Normally I'd ask to split features into a separate PR, but I think it's fine so long as we rename your PR accordingly.

Reply:
Renamed the PR 👍